### DISCUSSION TITLE
Battery production sensor unclear

### DISCUSSION THREAD

**Discussion by siegfried2003** [fsaris/home-assistant-zonneplan-one#151] [Q&A] [answered]
I everybody, this is my first question on github so don't be to hard :)
I have integrated my zonneplan battery into homeassistant including the p1 sensor. It works perfectly however the sensor thuisbatterij delivery is unclear to me. Yesterday we had day without solar production and our battery is set to self consumption, furthermore the battery was empty. however trough out the day the sensor batterij delivery went to 2 kwh in a straight line without the battery percentage changing. meanwhile the p1 sensor says the total usage of the grid is 12 kwh (this is correct) but where is the 2kwh of energy towards the battery going? Is it electricity from the grid that goes to the battery and out again just to keep the system online? 

<img width="807" alt="Scherm­afbeelding 2024-11-28 om 13 11 40" src="https://github.com/user-attachments/assets/19aed584-3de3-4839-a2d4-25be4201e549">

**Comment by fsaris** [comment]
The values should be equal as the values shown in the app. What can you see there?

**Reply by siegfried2003** [reply]
in the app it said nothing is was in the early days right after the install. I think its standby usage maybe. Otherwise it is a charge that is redirected towards homegrid and used elsewhere.

**Reply by fsaris** [reply]
Is it still incorrect or okay now?

**Reply by siegfried2003** [reply, marked as answer]
Hi fsaris, yes works fine. I indeed found out is the standby usage of the system. I am just surprised it uses so much electricity just to be power on. Thank u for the help!
